### PR TITLE
Migrate snackbar to toast

### PR DESCRIPTION
**Pull request by paphonb**
Screen Shot 2564-12-15 at 21 03 10

## Demo
[https://dev.cugetreg.com](https://dev.cugetreg.com)

## Checklist
- [x] Deploy a demo

### CODE REVIEW

**Comment by natTP** [comment]
image
- Can the icon be red instead?
- Can there also be a snackbar when deleting subject from cart in Course Schedule page

**Comment by paphonb** [comment]
> image
> 
> * Can the icon be red instead?
> * Can there also be a snackbar when deleting subject from cart in Course Schedule page

Screen Shot 2564-12-15 at 20 46 27

Now with an undo button

File: src/modules/App/components/CustomToaster/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+export function CustomToaster() {
+  const theme = useTheme()
+
+  return (
+    <ClassNames>
+      {({ css }) => (
+        <Toaster
+          position="top-center"
+          toastOptions={{
+            duration: 99999,
```
Comment (Anon-136) [inline src/modules/App/components/CustomToaster/index.tsx:30]: Why does default duration need to be Infinity?

**Review by Anon-136 — APPROVED** [review]
All good!

File: src/modules/App/components/CustomToaster/index.tsx (hunk 1 above)
Comment (paphonb) [reply src/modules/App/components/CustomToaster/index.tsx:30]: Oops that was for debugging

**Comment by Anon-136** [comment]
anyway, can we create a custom toast that wrap content with `ToastLayout` so that we do not have to wrap it ourselves?

File: src/modules/App/components/CustomToaster/index.tsx (hunk 1 above)
Comment (paphonb) [reply src/modules/App/components/CustomToaster/index.tsx:30]:
```suggestion
```

**Pull request merged**